src/server/status/index.js

```diff
@@ -53,6 +56,9 @@ export default function (kbnServer, server, config) {
   server.route(wrapAuth({
     method: 'GET',
     path: '/status',
+    config: {
+      tags: ['api']
```

Comment:
Is this because I tested using development mode, which has this proxy thing? I'm very much an ignorant when it comes to node.js/hapi/etc